lib/services.template.ejs

```diff
@@ -285,6 +285,17 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
 <%    }); // forEach methods name -%>
 <% } // for each scope -%>
 
+<% if (meta.modelSchema) { -%>
+        /**
+         * @ngdoc object
+         * @name <%-: moduleName %>.<%- modelName %>#schema
+         * @propertyOf <%-: moduleName %>.<%- modelName %>
+         * @description
+         * The schema of the model represented by this $resource
+         */
+        R.schema = <%- JSON.stringify(meta.modelSchema, null, 2) -%>;
```

Comment:
Actually, on the second thought, I think it's better to explicitly call `JSON.stringify` here. The intention of the `q` filter is to produce a quoted string. The fact that it calls JSON.stringify under the hood is an implementation detail that we should not rely on, as it may change in the future (in theory).